### PR TITLE
LPD-17834 Use the compact modal for the creation of the Display Page folders and  show the necessary information in the Folder Info Panel

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
>[!IMPORTANT]
>You need the FF LPS-189856

## Motivation

[Link to LPD-17833](https://liferay.atlassian.net/browse/LPD-17833)
[Link to LPD-17834](https://liferay.atlassian.net/browse/LPD-17834)

### Changes

Change the modal for the creation of DPT folders to a small size:
image

And add a button to manage permissions from the info panel:
image

## Test scenario 

1. Go to Design > Page Templates > Display Page Templates.
2. Create a folder (check the size of the modal).
3. Create a Display Page Template.
4. Use the Info Panel with both (folder and DPT) and check the button behaviour.

https://github.com/liferay-echo/liferay-portal/assets/93203423/6657c183-3908-4e9e-99e8-1e3c78175e62

>[!NOTE]
>The description input will be move to a different story so it won't be needed to check it in this pr. And this changes not need to add the FF due to the view is not show until it is activated.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-page-template-admin-web/types/src/main/resources/META-INF/resources/js/index.d.ts [hunk 1]
```diff
@@ -17,3 +17,4 @@ export {default as MasterLayoutDropdownPropsTransformer} from './propsTransforme
 export {default as MasterLayoutManagementToolbarPropsTransformer} from './propsTransformers/MasterLayoutManagementToolbarPropsTransformer';
 export {default as SelectDisplayPageMasterLayoutVerticalCardPropsTransformer} from './propsTransformers/SelectDisplayPageMasterLayoutVerticalCardPropsTransformer';
 export {default as SelectLayoutPageTemplateEntryMasterLayoutVerticalCardPropsTransformer} from './propsTransformers/SelectLayoutPageTemplateEntryMasterLayoutVerticalCardPropsTransformer';
+export {default as ManagePermissionsPropsTransformer} from './propsTransformers/ManagePermissionsPropsTransformer';
```
Comment (ealonso) [inline modules/apps/layout/layout-page-template-admin-web/types/src/main/resources/META-INF/resources/js/index.d.ts:20]: @BarbaraCabrera you should order it alphabetically

File: modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/js/index.ts [hunk 2]
```diff
@@ -56,3 +56,7 @@ export {default as SelectDisplayPageMasterLayoutVerticalCardPropsTransformer} fr
 // @ts-ignore
 
 export {default as SelectLayoutPageTemplateEntryMasterLayoutVerticalCardPropsTransformer} from './propsTransformers/SelectLayoutPageTemplateEntryMasterLayoutVerticalCardPropsTransformer';
+
+// @ts-ignore
+
+export {default as ManagePermissionsPropsTransformer} from './propsTransformers/ManagePermissionsPropsTransformer';
```
Comment (ealonso) [inline modules/apps/layout/layout-page-template-admin-web/src/main/resources/META-INF/resources/js/index.ts:62]: @BarbaraCabrera you should order it alphabetically

File: modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/display/context/DisplayPageTemplateInfoPanelDisplayContext.java [hunk 3]
```diff
@@ -140,6 +142,31 @@ public List<LayoutPageTemplateEntry> getLayoutPageTemplateEntries() {
...
+		return PermissionsURLTag.doTag(
+			StringPool.BLANK, LayoutPageTemplateCollection.class.getName(),
+			layoutPageTemplateCollection.getName(), null,
+			String.valueOf(
+				layoutPageTemplateCollection.
+					getLayoutPageTemplateCollectionId()),
+			LiferayWindowState.POP_UP.toString(), null, _httpServletRequest);
+	}
+
+	public String getPermissionsDisplayPageURL(
```
Comment (ealonso) [inline modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/display/context/DisplayPageTemplateInfoPanelDisplayContext.java:158]: @BarbaraCabrera name should be something like getPermissionsLayoutPageTemplateEntryURL the same for LayoutPageTemplateCollection

**Comment by victorg1991** [comment]
ci:forward

**Comment by BarbaraCabrera** [comment]
ci:forward:force

**Pull request closed without merging**